src/test/java/com/conveyal/datatools/manager/jobs/validation/RouteTypeValidatorBuilderTest.java

```diff
@@ -0,0 +1,43 @@
+package com.conveyal.datatools.manager.jobs.validation;
+
+import com.conveyal.datatools.DatatoolsTest;
+import com.conveyal.datatools.UnitTest;
+import com.conveyal.gtfs.validator.RouteTypeValidator;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class RouteTypeValidatorBuilderTest extends UnitTest {
+    @BeforeAll
+    static void setUp() throws IOException {
+        // start server if it isn't already running
```

Comment:
Full stop.

Reply:
Updated in https://github.com/ibi-group/datatools-server/pull/453/commits/c2d0ec01d7f9e1e6a4d54c08429d79ae6b06317a.